lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1061,7 +1061,7 @@ def restart_services(self, services, wait_time=0):
 
     def restart_mme(self, wait_time=20):
         """
-        Restart MME service and wait for the service to come up properly
+        Restart MME service
```

Comment:
We are still waiting for the services to come-up after restart inside this function in line number 1073. Is this change needed?